Index: seabios-1.8.2/Makefile
===================================================================
--- seabios-1.8.2.orig/Makefile
+++ seabios-1.8.2/Makefile
@@ -57,6 +57,7 @@ COMMONCFLAGS := -I$(OUT) -Isrc -Os -MD -
     $(call cc-option,$(CC),-Wtype-limits,) \
     -m32 -march=i386 -mregparm=3 -mpreferred-stack-boundary=2 \
     -minline-all-stringops -fomit-frame-pointer \
+    -fgnu89-inline \
     -freg-struct-return -ffreestanding -fno-delete-null-pointer-checks \
     -ffunction-sections -fdata-sections -fno-common -fno-merge-constants
 COMMONCFLAGS += $(call cc-option,$(CC),-nopie,)
